nfsd/nfsd.config

```diff
@@ -0,0 +1,13 @@
+CONFIG_NFSD=m
```

Comment:
I think we can simplify this a bit, as this is an in-tree kernel module, and build it as part of the `kernel` build, and then extract it as an extension

I cant take a stab at it